installer/config.yml

```diff
@@ -317,6 +317,47 @@ stream:
         # > default is false
         mobile: false
 
+    hls:
+      - # Whether the output is enabled.
+        # > default is false
+        enabled: false
+        # > must be one of ('mpegts', 'mp3', 'adts', 'mp4')
+        # > this field is REQUIRED
+        format: mpegts
+        # > segment_duration  (default:2.0)
+        segment_duration: 2.0
+        # > segments count  (default:5)
+        segment_count: 5
+        # > segments_overhead (default:5)
+        segments_overhead: 5
+        # Output public url, If not defined, the value will be generated from
+        # the [general.public_url] hostname, the output port and mount.
+        public_url:
+        # hls mount point
+        # > this field is REQUIRED
+        mount: hls/main
+        streams:
+          - # > prefix of generated fragment
+            # > this field is REQUIRED
+            fragment_prefix: mp3_low
+            # > codec must be one of ( 'libmp3lame', 'flac', 'aac', 'libopus', 'libvorbis')
```

Comment:
Let's not diverge from the spec: https://datatracker.ietf.org/doc/html/rfc8216#section-3.4